resource/css/skosmos.css

```diff
@@ -192,49 +196,180 @@ body {
     font-family: var(--font-family-heading);
   }
 
-  #search-multi-vocab-title {
-    color: var(--vocab-text);
+  #global-search-toggle {
+    transition: none;
+    background-color: var(--headerbar-vocab-bg);
```

Comment:
No such variable has been defined (but I see that it had been used before in skosmos.css line 173)
Just set the variable at the top?